src/glue/trigger.ts

```diff
@@ -46,13 +83,34 @@ export class Trigger extends Resource {
   private readonly _predicates: ITriggerPredicate[] = [];
 
   // Input properties
-  public readonly description?: string;
-  public readonly name?: string;
-  public readonly predicateOperator: PredicateOperator;
-  public readonly schedule?: Schedule;
-  public readonly startOnCreation?: boolean;
-  public readonly type: TriggerType;
-  public readonly workflow?: Workflow;
+  /**
```

Comment:
Indentation looks off.

Reply:
Tabs vs Spaces issue. Should be fixed now